### PR TITLE
Adding FunctionalPlus

### PR DESCRIPTION
**Pull request by Dobiasd**

### CODE REVIEW

File: recipes/FunctionalPlus/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```
Comment (npavlovikj) [inline recipes/FunctionalPlus/build.sh]: If you are building this tool for Windows, you will need to have build script for Windows, named `bld.bat` as well, https://docs.conda.io/projects/conda-build/en/latest/resources/build-scripts.html.

File: recipes/FunctionalPlus/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Dobiasd/FunctionalPlus/archive/v{{ version }}-p0.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true
```
[the remark this answers is not in the document]
Comment (Dobiasd) [reply recipes/FunctionalPlus/meta.yaml]: Thanks a lot. I just changed the line to `skip: True # [win]`.

File: recipes/FunctionalPlus/build.sh (hunk 1 above)
Comment (Dobiasd) [reply recipes/FunctionalPlus/build.sh]: Thanks again. Just added it.

**Comment by Dobiasd** [comment]
@chrisburr Thanks a lot for the improvements and for merging. =)

**Pull request merged**